parsel/selector.py

```diff
@@ -3,13 +3,18 @@
 """
 
 import sys
-
+import warnings
```

Comment:
:lipstick: `import warnings` should be right after `import sys`, then an empty line (to separate built-in modules from package modules).